From 2d2530cd4250e1617c64b72a7ad9b8b3801ef40b Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Wed, 9 Jul 2014 18:51:15 +1000
Subject: [PATCH 01/15] Mythmedatalookup: set PYTHONPATH so mythmedatalookup
 can be started on its own

(cherry picked from commit ab82af34abe17faa927c7ce0b8a81a5fe365773c)
---
 mythtv/programs/mythmetadatalookup/main.cpp |   12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/mythtv/programs/mythmetadatalookup/main.cpp b/mythtv/programs/mythmetadatalookup/main.cpp
index 0878925..1da7dfa 100644
--- a/mythtv/programs/mythmetadatalookup/main.cpp
+++ b/mythtv/programs/mythmetadatalookup/main.cpp
@@ -8,6 +8,9 @@ using namespace std;
 // Qt headers
 #include <QCoreApplication>
 #include <QEventLoop>
+#ifdef Q_OS_MAC
+#include <QProcessEnvironment>
+#endif
 
 // libmyth headers
 #include "exitcodes.h"
@@ -78,6 +81,15 @@ int main(int argc, char *argv[])
     QCoreApplication a(argc, argv);
     QCoreApplication::setApplicationName(MYTH_APPNAME_MYTHMETADATALOOKUP);
 
+#ifdef Q_OS_MAC
+    QString path = QCoreApplication::applicationDirPath();
+    setenv("PYTHONPATH",
+           QString("%1/../Resources/lib/python2.6/site-packages:%2")
+           .arg(path)
+           .arg(QProcessEnvironment::systemEnvironment().value("PYTHONPATH"))
+           .toUtf8().constData(), 1);
+#endif
+
     int retval;
     if ((retval = cmdline.ConfigureLogging()) != GENERIC_EXIT_OK)
         return retval;
-- 
1.7.10.2

